server/datastore/mysql/hosts.go

```diff
@@ -1147,6 +1152,10 @@ WHERE
 		}
 	}
 
+	for i := range hosts {
+		hosts[i].DisplayName = fleet.HostDisplayName(hosts[i].ComputerName, hosts[i].HostName, hosts[i].HardwareModel, hosts[i].HardwareSerial)
+	}
+
```

Comment:
Through a fairly convoluted set of steps, this is what the "list hosts" endpoint ends up calling:

1. https://github.com/fleetdm/fleet/blob/72571a9f8e76c9b709df9396d753dba415b3743c/server/service/hosts.go#L142-L154
2. https://github.com/fleetdm/fleet/blob/347c65b5c672e7b199e3645c97a85e1b71701d85/server/fleet/hostresponse.go#L21-L36
3. https://github.com/fleetdm/fleet/blob/134c74a94b0025e700627c1fb0699f69788aea88/server/fleet/hosts.go#L849-L851